ios-xcframework/Podfile

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Note that the pods in this array might seem unused if you look for
+# `import` statements in this codebase. However, make sure to also check
+# whether they are used in the gutenberg-mobile and Gutenberg projects.
+#
+# See https://github.com/wordpress-mobile/gutenberg-mobile/issues/5025
+DEPENDENCIES = %w[
+  FBLazyVector
+  React
+  ReactCommon
+  RCTRequired
+  RCTTypeSafety
+  React-Core
+  React-CoreModules
+  React-RCTActionSheet
+  React-RCTAnimation
+  React-RCTBlob
+  React-RCTImage
+  React-RCTLinking
+  React-RCTNetwork
+  React-RCTSettings
+  React-RCTText
+  React-RCTVibration
+  React-callinvoker
+  React-cxxreact
+  React-jsinspector
+  React-jsi
+  React-jsiexecutor
+  React-logger
+  React-perflogger
+  React-runtimeexecutor
+  boost
+  Yoga
+  RCT-Folly
+  glog
+  react-native-safe-area
+  react-native-safe-area-context
+  react-native-video
+  react-native-webview
+  RNSVG
+  react-native-slider
+  BVLinearGradient
+  react-native-get-random-values
+  react-native-blur
+  RNScreens
+  RNReanimated
+  RNGestureHandler
+  RNCMaskedView
+  RNCClipboard
+  RNFastImage
+  React-Codegen
+  React-bridging
+].freeze
+
+def gutenberg_dependencies
+  podspec_prefix = '..'
+
+  # FBReactNativeSpec needs special treatment because of react-native-codegen code generation
+  pod 'FBReactNativeSpec',
+      podspec: "#{podspec_prefix}/third-party-podspecs/FBReactNativeSpec/FBReactNativeSpec.podspec.json"
+
+  DEPENDENCIES.each do |pod_name|
+    pod pod_name, podspec: "#{podspec_prefix}/third-party-podspecs/#{pod_name}.podspec.json"
+  end
```

Comment:
Question: it appears these `DEPENDENCIES` podspec files are generated from [generate-podspecs.sh](https://github.com/wordpress-mobile/gutenberg-mobile/blob/1c64b735fc15ea47b64d3be447397e0ebc55d230/bin/generate-podspecs.sh#L76). Should we run that script before running `pod install` against this Podfile?

Reply:
Good question 🤔 

The outcome of running that script is tracked in the repo, and I think it's meant to run only when the React Native version changes, which is infrequently. So I'd say it's not really necessary. 

Having said that, it shouldn't hurt to run the script anyway, just in case?

Keen to hear what the folks more familiar with the Gutenberg setup think.

Reply:
One thing to remember if we go down that route is that at the moment the script is interactive:

https://github.com/wordpress-mobile/gutenberg-mobile/blob/1c64b735fc15ea47b64d3be447397e0ebc55d230/bin/generate-podspecs.sh#L29

We'll want to add an option to bypass that prompt.

Reply:
Actually, thinking about it, with the current setup running the script in CI would not work because of the lack of node modules 😞

Reply:
I'm not familiar with this repo at all, not to mention the release process. Please feel free to ignore this comment. But if I were to guess, all the podspecs in `third-party-podspecs` are dependecies of the `Gutenberg` pod, correct? If that's the case, we could just find all the podspec in the folder and use them add them as `pod ...`, instead of having a hard-coded list of dependencies?

Reply:
> all the podspecs in `third-party-podspecs` are dependecies of the `Gutenberg` pod, correct? If that's the case, we could just find all the podspec in the folder and use them add them as `pod ...,` instead of having a hard-coded list of dependencies?

I think that would work. I'll try it as a followup PR. It'd be really cool to remove that long list! 🙇‍♂️